src/output.rs

```diff
@@ -141,3 +141,12 @@ fn create_cooldown_sentinel(path: PathBuf) -> Result<(), Error> {
         .open(path)?;
     Ok(())
 }
+
+fn should_print_activation() -> bool {
+    let configured_to_print: bool;
+    match env::var("SHADOWENV_SILENT") {
+        Ok(_) => configured_to_print = false,
```

Comment:
In the `Ok` case, I'd love to see values of `"0"`, `"false"`, `"no"`, and `""` also return `true`.